ubi/LICENSE

```diff
@@ -0,0 +1,354 @@
+Mozilla Public License, version 2.0
```

Comment:
This is what Vault does too. Need to copy the LICENSE into this folder so that Docker build can copy it into the container.